src/linux_syscall_support_unittest.cc

```diff
@@ -140,42 +140,73 @@ static void SigAction(int signum, siginfo_t *si, void *arg) {
 
 static void Sigaction() {
   puts("Sigaction...");
+  #if defined(__aarch64__)
+  const size_t kSigsetSize = sizeof(struct kernel_sigset_t);
+  #endif
   int signum       = SIGPWR;
   for (int info = 0; info < 2; info++) {
     signaled         = 0;
     struct kernel_sigaction sa = ZERO_SIGACT, old, orig;
+    #if defined(__aarch64__)
+    CHECK(!sys_rt_sigaction(signum, NULL, &orig, kSigsetSize));
+    #else
     CHECK(!sys_sigaction(signum, NULL, &orig));
+    #endif
```

Comment:
Is there a reason we can't just switch all the architectures to use rt_sigaction()?  It should work on Intel shouldn't it?

Reply:
`linux_syscall_support_unittest` fails with SIGSEGV when calling `sys_rt_sigsuspend()` on Intel.
And it looks like `rt` functions can not be mixed with non `rt`.